Automattic/jetpack
# [Jetpack AI] Store post ids

Fixes #28444

## Proposed changes:
* Pass post_id to the API
* Trigger appropriate tracks events
* Store number of JetpackAI usages in post in post meta



## Does this pull request change what data or activity we track or use?

Store post ids and the number of uses for Jetpack AI

## Testing instructions:
* `jetpack build` && `jetpack rsync` to a WPCOM Simple site as this introduces changes on the sandbox as well
* Trigger GPT block on the frontend,
* Inspect the network tab
* See post_id now in the request params
* Inspect `t.gif` calls for tracks events
* Search for `feature:openai` in Kibana to see post_id now present in Logstash logs
* Also, after some time you should see the events in tracks
* You can verify the meta with `wp post meta get 646 _jetpack_ai_calls --url=` 
 
<img width="860" alt="Zrzut ekranu 2023-01-20 o 13 06 39" src="https://user-images.githubusercontent.com/3775068/213691511-2785b6c8-97a0-4c6c-9ae6-5a3df1b89aa6.png">

<img width="476" alt="Zrzut ekranu 2023-01-20 o 12 31 07" src="https://user-images.githubusercontent.com/3775068/213688136-8d5c2fd7-1a4d-472c-90da-32dcb3ae2a49.png">
